### PR TITLE
Added Todolist Cli-Version

### LINKED ISSUE #871: Todo-list | File handling | CLI
I want to create a Todo list feature which will work based on files-handling
please assign me for this issue under gssoc'21

### PR DESCRIPTION
**Pull request by Avishake007**
# Description
Here I have created todolist in cli version
## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: TODO (CLI-VER)/todolist.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
+#Check for the existence of file
+no_of_items=0
+try:
+    f=open("todolist.txt")
```
Comment (kaustubhgupta) [inline TODO (CLI-VER)/todolist.py:4]:
```suggestion
    f=open("./TODO (CLI-VER)/todolist.txt")
```

File: TODO (CLI-VER)/todolist.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        with open('todolist.txt') as f:
+            for i in f.readlines():
+                print(i)
+                todo.append(i)
+        print("____________________________________________________________")
+    print("Press 1: Add Item \nPress 2: Delete the Item\nPress 3: Update item\nPress 4:Display Items\nPress 5:Exit")
+    n=int(input())
+        
+
+
```
Comment (kaustubhgupta) [inline TODO (CLI-VER)/todolist.py:119]: Remove one blank line

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: TODO (CLI-VER)/todolist.py (hunk 1 above)
Comment (kaustubhgupta) [reply TODO (CLI-VER)/todolist.py:4]: This is left

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Code works perfectly, just this left:

**Review by kaustubhgupta — APPROVED** [review]
💯

**Review by Kushal997-das — APPROVED** [review]
LGTM 👍

**Review by santushtisharma10 — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**